### PR TITLE
Accept more type of collaborator as reviewers

### PR DESCRIPTION
**Pull request by sileht**
The engine v1 was accepting any kind of collaborators.

This change adds "contributor" to engine v2.

### CODE REVIEW

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 1bb683e7d2b16484af94280db828c830282b6642

**Comment by sileht** [comment]
Pull-request updated, HEAD is now d170f00300e18bc84aa466a7bb9abf29995c0200

File: doc/source/conditions.rst [hunk 1]
```diff
@@ -166,6 +166,12 @@ Operators List
        the ``#`` prefix.
 
 
+.. note::
+
+    For all `reviewers` related attributes, only users with `write` or `admin`
+    permission on the repository are used, other are ignored.
```
Comment (jd) [inline doc/source/conditions.rst]: others

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 28ced49b4cfff8c4e714ab95ede07856ffdb91de

**Comment by sileht** [comment]
Pull-request updated, HEAD is now f00fffa33bd09848a5ba9d90c3fe038c92a3f8a1

**Comment by sileht** [comment]
Pull-request updated, HEAD is now 43b48894ac42345ebf9e8011eae0f9075c513a3f

**Pull request merged**